src/frontend/web/src/ratingpage/Ratingpage.js

```diff
@@ -0,0 +1,142 @@
+    this.likeCard = this.likeCard.bind(this);
+    this.dislikeCard = this.dislikeCard.bind(this);
+
+    this.state = {
+      visible_like: false,
+      visible_dislike: false
+    };
+  };
+
+  likeCard() {
+    this.flashLike();
+    // Wait until the flash animation disapears.
+    setTimeout(() => {this.slider.slickPrev()} , 900);
+  };
+
+  dislikeCard() {
+    this.flashDislike();
+    // Wait until the flash animation disapears.
+    setTimeout(() => {this.slider.slickNext()}, 900);
+  };
+
+  onKeyDown = (event) => {
+      if(event.key === 'ArrowLeft'){
+        this.dislikeCard();
+      };
+
+      if(event.key === 'ArrowRight') {
+        this.likeCard();
+      };
+  };
+
+  componentDidMount() {
+      document.addEventListener('keydown', this.onKeyDown);
+  };
+
+  componentWillUnmount(){
+    document.removeEventListener('keydown',this.onKeyDown);
+  };
+
+  flashLike = () => {
+    this.setState({ visible_like: true });
+    setTimeout(() => {
+      this.setState({ visible_like: false });
+    }, 450);
+  };
+
+  flashDislike = () => {
+    this.setState({ visible_dislike: true });
+    setTimeout(() => {
+      this.setState({ visible_dislike: false });
+    }, 450);
+  };
+
+  render (){
+    const settings = {
+      infinite: true,
+      speed: 800,
+      slidesToShow: 1,
+      slidesToScroll: 1,
+      arrows: false,
+      draggable: false
+    };
+
+    return (
+      <Layout className='layout'>
+        <div className='rating-wrapper' dir='rtl'>
+          <div className='rating-center'>
+            <div className='slider-wrapper'>
+              <Slider ref={c => (this.slider = c)} {...settings}>
+                <RatingCard />
+                <RatingCard />
+              </Slider>
+            </div>
+
+            <div className='button-section left-section' >
+            <Button id='dislike-btn' onClick={this.dislikeCard} type='primary'
+             shape='circle'size='large'>
+              <DislikeOutlined style={{ fontSize: '7vh' }}/>
+             </Button>
+            </div>
+
+            <div className='flash-wrapper left-section'>
+              <div className={`hidden-content${this.state.visible_dislike ?
+                 ' visible-content' : ''}`}>
+                <div className='flash-content dislike-animation'>פחות</div>
+              </div>
+            </div>
+
+            <div className='button-section right-section'>
+            <Button id='like-btn' onClick={this.likeCard} type='primary'
+             shape='circle'size='large'>
+              <LikeOutlined style={{ fontSize: '7vh' }}/>
+             </Button>
+            </div>
+
+            <div className='flash-wrapper right-section'>
+              <div className={`hidden-content${this.state.visible_like ?
+                 ' visible-content' : ''}`}>
+                <div className='flash-content like-animation'>אהבתי!</div>
+              </div>
+            </div>
+
+            {/* Arrows image taken from here:
```

Comment:
It looks a bit not related to the website style.

Reply:
Yup I not sure about this as well. Is it going to be clear enough these images refer to the keyboard’s arrows, if we use for example ant’s icons? 
![image](https://user-images.githubusercontent.com/35609587/77853759-86721500-71ee-11ea-863e-719df8f15f5f.png)
I don’t want them to be mistaken by the user for some buttons with an arrow on them

Reply:
I think it is clear enough if these icons don't have a border. @omer-moyal @mryuv1 - what do you think?
Maybe use the filled one or the two tone one?
![image](https://user-images.githubusercontent.com/44426592/77856472-1c626b80-7200-11ea-8dac-984a6705f59b.png)

Reply:
its indeed dont fit with the web style.
if we remain the titel "use keybord arrow", we dont neccerly need the arrow icon, i think people will understand. 
but if we do use the icons, i think its better to use an arrow with a border. I liked the last blue one.
any way, we shoud make the icons simmetric related the middel.

Reply:
In my opinion it's clear enough,
 maybe we can put it in a box and give it an opacity of 0.2

Reply:
Ok so it does look clear :)
I’m leaving it with this blue until we decide on the color palette for the website.
Also, what do you think about centering the arrows, or removing the icons as Omer suggested?

option 1:
![text_small](https://user-images.githubusercontent.com/35609587/77928264-df09e680-72b0-11ea-9343-4a50fb89cd67.png)

option 2:
![no_text_small](https://user-images.githubusercontent.com/35609587/77928296-e9c47b80-72b0-11ea-93f3-9fd0ad5c5eae.png)

option 3:
![new_text_small](https://user-images.githubusercontent.com/35609587/77928350-f8ab2e00-72b0-11ea-8637-7d7f674da1ce.png)

Reply:
Thumb rule: A UX that has to be documented is not a good UX.
I think that we either have to agree that this is clear enough even if we don't write anything (and I think it is),  or (a cute solution) you could add ant's Tooltip on the buttons.

Reply:
I like the tooltip idea! Will fix it.